src/modules/launcher/PowerLauncher/SettingsReader.cs

```diff
@@ -215,5 +216,37 @@ private static IEnumerable<PowerLauncherPluginSettings> GetDefaultPluginsSetting
                 AdditionalOptions = x.Plugin is ISettingProvider ? (x.Plugin as ISettingProvider).AdditionalOptions : new List<PluginAdditionalOption>(),
             });
         }
+
+        private void UpdateAdditionalOptionsSettings(PowerLauncherSettings settings)
+        {
+            foreach (var pluginPair in PluginManager.AllPlugins)
+            {
+                var plugin = pluginPair.Plugin;
+
+                if (plugin is ISettingProvider && (plugin as ISettingProvider).AdditionalOptions != null)
+                {
+                    var pluginSettings = settings.Plugins.Single(p => p.Id == pluginPair.Metadata.ID);
```

Comment:
Let's change it to SingleOrDefault. It is possible that newly added plugin is not in user settings yet.